### DISCUSSION TITLE
help- I could not install the package

### DISCUSSION THREAD

**Discussion by chandagsin** [deepmodeling/dpdata#406] [Q&A]
Dear Developers, 

I am a new python user. I used the standard installation procedure described in the README file, 
     git clone https://github.com/deepmodeling/dpdata.git dpdata
   (This was done successfully.)
     cd dpdata
     python setup.py install
  [Errno 2] No such file or directory. 

I checked the package but did not find a file name "setup.py". I have tried online/offline pip/pip3 method but failed. 

My question is, do I need to create a file "setup.py"? 
thanks. 
CDS.

**Comment by HuangJiameng** [comment]
use `pip install .` instead.

**Reply by chandagsin** [reply]
Dear HuangJiameng, 
Thanks for your reply. I used pip install dpdata. the system reported "Defaulting to user installation because normal site-packages is not writeable". I am not a root user - I do not have enough permission. Better to have setup.py or whl for installation. 
thx. CDS

**Reply by HuangJiameng** [reply]
try `pip install --user <package-name>`
